e2e-tests/admin/tests/scope.spec.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { test } from '../../global-setup.js';
+import { expect } from '@playwright/test';
+import { OrgsPage } from '../pages/orgs.js';
+
+import * as boundaryHttp from '../../helpers/boundary-http.js';
+
+test(
+    'Session picker',
+    { tag: ['@ce', '@ent', '@aws', '@docker'] },
+    async ({ page, request }) => {
+        let org;
+        try {
+            //Create an org and project attached to the org
+            org = await boundaryHttp.createOrg(request);
+            const project = await boundaryHttp.createProject(request, org.id);
+
+            //Choose the org from the scope picker
+            await page.goto('/');
+            const orgsPage = new OrgsPage(page);
+            await orgsPage.chooseScopeFromDropdown(org.name);
+
+            //Expect the project to be displayed on the projects page
+            await expect(
+                page.getByRole('link', { name: project.name })
+            ).toBeVisible()
+
+            //Switch back to global scope
+            await orgsPage.chooseScopeFromDropdown("Global");
```

Comment:
nit: I thought the convention was to use single quotes here, but the linter didn't complain, so 🤷 ?
```suggestion
            await orgsPage.chooseScopeFromDropdown('Global');
```